COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@kernc I'm not what makes you the expert on my terms of employment but the
top comment might clear things up for you:
https://news.ycombinator.com/item?id=1969979
On Dec 8, 2015 11:38 AM, "kernc" <notifications@github.com> wrote:

> In COPYING
> <https://github.com/NixOS/nixpkgs/pull/11535#discussion_r46977676>:
>
> > @@ -1,4 +1,4 @@
> > -Copyright (c) 2003-2006 Eelco Dolstra
> > +Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
>
> @eduarrrd <https://github.com/eduarrrd> Google does *not* hold the
> copyright to patches you contribute at your home in your free time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/11535/files#r46977676>.
>